source/docs.js

```diff
@@ -7,6 +7,6 @@
 // Demonstrate BS3 Tooltips
 $(function () {
     $('[data-toggle="tooltip"]').tooltip({
-        container: 'body'
+        container: 'body',
```

Comment:
So, `always-multiline` requires this comma. I'm not a fan of adding a comma that isn't necessary, so I'd suggest changing the configuration of "comma-dangle" to `only-multiline` so that dangling commas _can be used_, but _aren't required_.

@Droogans, any thoughts?